internal/civisibility/integrations/gotesting/testcontroller_test.go

```diff
@@ -43,13 +44,16 @@ func TestMain(m *testing.M) {
 		runEarlyFlakyTestDetectionTests(m)
 	} else if internal.BoolEnv(scenarios[2], false) {
 		fmt.Printf("Scenario %s started.\n", scenarios[2])
-		runFlakyTestRetriesWithEarlyFlakyTestDetectionTests(m)
+		runFlakyTestRetriesWithEarlyFlakyTestDetectionTests(m, false)
 	} else if internal.BoolEnv(scenarios[3], false) {
 		fmt.Printf("Scenario %s started.\n", scenarios[3])
 		runIntelligentTestRunnerTests(m)
 	} else if internal.BoolEnv(scenarios[4], false) {
 		fmt.Printf("Scenario %s started.\n", scenarios[4])
 		runTestManagementTests(m)
+	} else if internal.BoolEnv(scenarios[5], false) {
+		fmt.Printf("Scenario %s started.\n", scenarios[5])
```

Comment:
NIT: define and reuse the same string, maybe?

Reply:
done in [a2a2a04](https://github.com/DataDog/dd-trace-go/pull/3389/commits/a2a2a04f38eebca0a1cd7aa5922549a19876546e)